docs/modules/ROOT/pages/explanations/decisions/mongodb.adoc

```diff
@@ -0,0 +1,37 @@
+= MongoDB
+
+== Problem
+
+We want to Provide MongoDB via AppCat.
+But their licensing might be an issue.
+This decision looks closely at the licensing and alternative products that can be used as drop-in replacements for MongoDB.
+
+== Proposals
+
+Proposal 1: Use MongoDB Community Edition::
+MongoDB community edition is licensed under the https://en.wikipedia.org/wiki/Server_Side_Public_License[SSPL].
+The SSPL dictates that the source code for the tooling that provides the services, needs to be available to the public as well.
+However, it would also need to be licensed under the SSPL.
+
+Proposal 2: Use MongoDB Enterprise Operator::
+MongoDB Enterprise Operator can be used to manage MongoDB instances on Kubernetes.
+The licensing allows to use it as a SaaS according to their docs.
+There's a https://github.com/mongodb/helm-charts/tree/main/charts/enterprise-operator[helm chart] to deploy it.
+It does need an https://www.mongodb.com/pricing[enterprise license] though.
```

Comment:
How does this work? Is this enforced somehow? Do we need to distribute MongoDB license information to all customer clusters?

Reply:
Yes the operator needs a license key. Not sure if the operator needs to have an outbound connection to a licensing server though.

Also, as far as I know the first business case we have for MongoDB is for Exoscale only, so it would only be deployed for Exoscale in the beginning.